docs/guides/events.md

```diff
@@ -1,6 +1,6 @@
 # Events
 
-An event is what causes a state machine to [transition](./transitions.md) from its current [state](./states.md) to its next state. All state transitions in a state machine are due to these events; state cannot change unless some stimulus (the event) causes it to change.
+An event is what causes a state machine to [transition](./transitions.md) from its current [state](./states.md) to its next state. All state transitions in a state machine are due to these events; a state cannot change unless some stimulus (the event) causes it to change.
```

Comment:
```suggestion
An event is what causes a state machine to [transition](./transitions.md) from its current [state](./states.md) to its next state. All state transitions in a state machine are due to these events; state cannot change unless some stimulus (the event) causes it to change.
```

This makes more sense in the original form imo

Reply:
I prefer the version with "a," but am not a native speaker, so 🤷